Godeps/_workspace/src/github.com/openshift/openshift-sdn/pkg/ovssubnet/controller/multitenant/bin/openshift-sdn-multitenant-setup.sh

```diff
@@ -114,6 +120,7 @@ function setup() {
     ip link set vovsbr up
     ip link set vlinuxbr txqueuelen 0
     ip link set vovsbr txqueuelen 0
+    brctl addif lbr0 vlinuxbr
```

Comment:
Maybe I didnt get the logic right. What if lbr0 gets deleted by chance and the admin restarts the sdn setup?
What I see is that we will have docker_network_config called with an update, but setup will be skipped. Which means we will report a valid SDN but we will still not have vlinuxbr added to the bridge, which means builds will fail.